unicorn-core/src/test/scala/org/virtuslab/unicorn/repositories/UsersRepositoryTest.scala

```diff
@@ -59,124 +60,143 @@ trait UsersRepositoryTest extends OptionValues {
         'lastName(user.lastName)
       )
       userOpt.value.id shouldBe defined
+    }
   }
 
-  it should "save and query multiple users" in rollback {
-    implicit session =>
-      // setup
-      UsersRepository.create()
+  it should "save and query multiple users" in runWithRollback {
+    val users = (Stream from 1 take 10) map (n => UserRow(None, "test@email.com", "Krzysztof" + n, "Nowak"))
```

Comment:
We call `slick.dbio.DBIO#sequence` in our implementation of `saveAll` and `sequence` method checks the length of passed collection. I wonder if this is a bug or a feature..
Anyway let's not depend of Slick's internal implementation and do it safe with `toIndexedSeq` call